docker/vendor/fluentd/Gemfile

```diff
@@ -0,0 +1,12 @@
+source 'https://rubygems.org'
+
+gem 'activesupport', '~>5.2.3'
+gem 'fluentd', '<=1.5.1'
+gem 'fluent-plugin-concat', '~>2.3.0'
+gem 'fluent-plugin-detect-exceptions', '~>0.0.12'
+gem 'fluent-plugin-elasticsearch', '~>3.5.2'
+gem 'fluent-plugin-kubernetes_metadata_filter', '~>2.2.0'
+gem 'fluent-plugin-multi-format-parser', '~>1.0.0'
+gem 'fluent-plugin-prometheus', '~>1.4.0'
+gem 'fluent-plugin-systemd', '~>1.0.2'
+gem 'oj', '~>3.7.12'
```

Comment:
we can move `RUN gem install fluent-plugin-rewrite-tag-filter` to Gemfile i think.